tests/integration/profile_update/test_bookmark_profile_update.py

```
@@ -0,0 +1,437 @@
+        """Redis 캐싱이 포함된 북마크 이벤트 처리 테스트"""
+        # Given
+        user_id = 123
+        bookmark_data = sample_bookmarks[0]
+
+        # Mock Redis 클라이언트
+        mock_redis = AsyncMock()
+        profile_processor.redis_client = mock_redis
+
+        with patch.object(profile_processor.openai_service, 'extract_keywords') as mock_keywords, \
+             patch.object(profile_processor.openai_service, 'create_embedding') as mock_embed, \
+             patch.object(mock_repositories['user_profile_repo'], 'update_profile') as mock_save:
```

Ruff (0.11.9)

Local variable `mock_save` is assigned to but never used

Remove assignment to unused variable `mock_save`

(F841)